azkaban-common/src/test/java/azkaban/project/DirectoryYamlFlowLoaderTest.java

```diff
@@ -56,131 +57,109 @@ public void testLoadBasicYamlFile() {
     final DirectoryYamlFlowLoader loader = new DirectoryYamlFlowLoader(new Props());
 
     loader.loadProjectFlow(this.project, ExecutionsTestUtil.getFlowDir(BASIC_FLOW_YAML_DIR));
-    Assert.assertEquals(0, loader.getErrors().size());
-    Assert.assertEquals(1, loader.getFlowMap().size());
-    Assert.assertTrue(loader.getFlowMap().containsKey(BASIC_FLOW_1));
+    assertThat(loader.getErrors().size()).isEqualTo(0);
```

Comment:
My previous suggestion was to consolidate these duplicate code since you always need to test loader.getFlowMap().size() in every @test method. Also, flow.getNodes().size(), flow.getErrors().size().  So, why not consolidate them into one method with a number of arguments.

Reply:
Sure, I just submitted the changes to consolidate the test case. Thanks for the comment!